wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -1834,7 +1835,40 @@ impl super::Adapter {
 
         let mem_allocator = {
             let limits = self.phd_capabilities.properties.limits;
-            let config = gpu_alloc::Config::i_am_prototyping(); //TODO
+            // TODO: These configuration options should take hardware capabilities
```

Comment:
Note: vulkan's allocator parameters don't map super well to the manual hints, however the plan is to switch to `gpu_allocator` (the one we use with d3d12), which better aligns.